lib/talon/resource.ex

```diff
@@ -167,7 +176,7 @@ defmodule Talon.Resource do
       def resource_paths(%{talon: talon} = _talon) do
         talon.resources()
         |> Enum.map(fn talon_resource ->
-          schema = talon_resource.schema()
+          schema = talon_resource.schema() # TODO: let resource determine presentation name (DJS)
```

Comment:
Yes, that's actually meant for the line below but the line I used was much shorter, so I cheated :)